exporter/datadogexporter/metrics_translator.go

```diff
@@ -39,9 +41,30 @@ func getTags(labels pdata.StringMap) []string {
 	return tags
 }
 
+// isCumulativeMonotonic checks if a metric is a cumulative monotonic metric
+func isCumulativeMonotonic(md pdata.Metric) bool {
+	switch md.DataType() {
+	case pdata.MetricDataTypeIntSum:
+		return md.IntSum().AggregationTemporality() == pdata.AggregationTemporalityCumulative &&
+			md.IntSum().IsMonotonic()
+	case pdata.MetricDataTypeDoubleSum:
+		return md.DoubleSum().AggregationTemporality() == pdata.AggregationTemporalityCumulative &&
+			md.DoubleSum().IsMonotonic()
+	}
+	return false
+}
+
+// metricDimensionsToMapKey maps name and tags to a string to use as an identifier
+// The tags order does not matter
+func metricDimensionsToMapKey(name string, tags []string) string {
+	const separator string = "}{" // These are invalid in tags
```

Comment:
I think it is rather unlikely that something bad happens here but this could be a problem if someone uses arbitrary values in tags. We could use some different separator if there is any that we know for sure can't be in tags (even arbitrary ones)